app/controllers/concerns/lti.rb

```diff
@@ -226,6 +226,7 @@ def store_lti_session_data(parameters)
     @lti_parameters.save!
 
     session[:external_user_id] = current_user.id
+    session[:pair_programming] = params[:custom_pair_programming] || false
```

Comment:
Since this concern should not have direct access to the parameters, use the variable passed as a parameter instead:

```suggestion
    session[:pair_programming] = parameters[:custom_pair_programming] || false
```

Reply:
Further, we can probably expect that we store this information in the session. Within the respective spec, add:

```rb
expect(controller.session).to receive(:[]=).with(:pair_programming, anything)
```